var/spack/repos/builtin/packages/python/package.py

```diff
@@ -842,6 +842,23 @@ def config_vars(self):
         Returns:
             dict: variable definitions
         """
+        # Some values set by sysconfig may not always exist on Windows, so
+        # compute Windows alternatives
+        def repair_win_sysconf(conf):
+            if not is_windows:
```

Comment:
I think it would be clearer to just have `if is_windows: .... `

Reply:
Sounds good!